packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -562,6 +562,7 @@ class _UiKitViewState extends State<UiKitView> {
   UiKitViewController? _controller;
   TextDirection? _layoutDirection;
   bool _initialized = false;
+  FocusNode? _focusNode;
```

Comment:
Is there a situation where _focusNode is nil and needs to be accessed?
Maybe `late FocusNode _focusNode;` is better here?